g4hepem.spec

```
@@ -0,0 +1,50 @@
+### RPM external g4hepem 20221014
+## INCLUDE compilation_flags
+%define tag %{realversion}
+%define branch master
+%define github_user mnovak42
+Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/v%{tag}&export=%{n}.%{realversion}&output=/%{n}.%{realversion}-%{tag}.tgz
```

@hahnjo , there is `v20221014` tag in the github repo, so either create `v20221014` tag or update the above line
```suggestion
Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}.%{realversion}&output=/%{n}.%{realversion}-%{tag}.tgz
```

---

Gnah, I didn't see this one...